Changed behaviour so that the command handler finished as soon as a m…

KarlLundengaard opened the pull request:
…atch is found

KarlLundengaard committed:
Changed behaviour so that the command handler finished as soon as a match is found

KarlLundengaard commented:
This update changes the behaviour of the handler in the base layer to "early-out" when a matching case if found, instead of finding all matches. This is done to mitigate performance issues with evaluation functions.

jarkabaker reviewed (approved):
Sorry Karl! I completely missed this PR.

KarlLundengaard committed:
Fixed bug when a case overlaps with the default case

If a case overlapped with the default case the default case now takes priority.
